src/documents/integrationIntro.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Preferences Server Integration
+layout: default
+category: Overview
+---
+
+The First Discovery Tool is integrated with the [GPII Preferences Server](https://github.com/GPII/universal/blob/master/documentation/PreferencesServer.md) to save user selected preferences. The users of The First Discovery Tool are usually new to the concept of preferences that can be used to customize a user interface to match their needs. They normally don't have an existing ID with the GPII Preferences Server. This integration saves user selected preferences to the GPII Preferences Server and the GPII Preferences Server returns a token that can be kept and used to retrieved the save preferences when needed. The returned token is essentially this user's newly-created user IDs with the GPII Preferences Server.
+
+*Note*: The First Discovery Tool potentially can be integrated with any preferences servers that support the save of user preferences. This document only explains how it integrates with the GPII Preferences Server.
```

Comment:
I looked thru the documentation of the first discovery tool and infusion, it seems we usually only bold the word "Note". Italicized words are used for certain words or sentences that really need to be emphasized in the notes. 

Let me know if you find different cases.